package.json

```diff
@@ -45,6 +46,7 @@
     "webpack-dev-server": "^4.11.1"
   },
   "dependencies": {
-    "json-rpc-2.0": "^1.6.0"
+    "json-rpc-2.0": "^1.6.0",
+    "uuid": "^10.0.0"
```

Comment:
Can remove `uuid` and `@types/uuid` and just use the `randomUUID` from helpers.

Reply:
Done!